stories/header-date-selector.stories.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
It'd be good to add the Site Kit header here. Eg:

<img width="772" alt="Screenshot 2020-11-18 at 13 52 30" src="https://user-images.githubusercontent.com/90871/99538750-4e9a2680-29a5-11eb-8814-18b585efda3c.png">